common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -3773,7 +3773,7 @@ MAV_CMD_DO_SET_CAM_TRIGG_DIST
 Supported by: Copter, Plane, Rover.
 
 Trigger the :ref:`camera shutter <common-camera-shutter-with-servo>` at
-regular distance intervals. This command is useful in :ref:`camera survey missions <common-camera-control-and-auto-missions-in-mission-planner>`.
+regular distance intervals. This command is useful in :ref:`camera survey missions <common-camera-control-and-auto-missions-in-mission-planner>`.  To trigger the camera once, immediatly after passing the DO command, set param3 to 1.
```

Comment:
immediately

Reply:
add note that this is from 4.1.x releases onward

Reply:
Opps! sorted that spelling mistake.
@tridge I have added a place holder for the release.  When it goes into Beta I shall update.
@Hwurzburg could you add the delay merge label for me please?

Reply:
Matt, it will be in 4.1....just a nit, but could it say "Trigger immediately parameter3 is available from ArduPilot firmware 4.1 onwards."
I will merge after that...no delay label needed

Reply:
Done as requested.

Reply:
Thanks Matt, merged!